api/app.py

```diff
@@ -1,19 +1,19 @@
 from fastapi import FastAPI
 from fastapi.responses import JSONResponse, PlainTextResponse
 from gradescopeClient import GradescopeClient
-import os
-import json 
```

Comment:
`os` and `json` are imported in `utils.py` and we are importing * from the `utils.py`. Apparently, because of this, all libraries imported in the `utils.py` file is accessible in `app.py`. (I did not think it would work this way until yesterday, but it makes sense when I think about it). Your choice as to whether you want to keep these imports in `app.py`.

Reply:
> Ah I see, amazing